Source/spells.cpp

```diff
@@ -405,15 +396,6 @@ int GetSpellStaffLevel(spell_id s)
 	if (!gbIsHellfire && s > SPL_LASTDIABLO)
 		return -1;
 
-	if (gbIsHellfire) {
-		switch (s) {
-		case SPL_ELEMENT:

```

Comment:
please add it to the spawn exception

Reply:
Did so. I tried finding a comprehensive list of changes between standard Diablo and the Spawn install, to check for other differences, but wasn't really able to find anything. Do you maybe have a link?